tests/integration/standard/test_metadata.py

```diff
@@ -1740,8 +1746,12 @@ def test_init_cond(self):
                 self.assertDictContainsSubset(expected_map_values, map_res)
                 init_not_updated = dict((k, init_cond[k]) for k in set(init_cond) - expected_key_set)
                 self.assertDictContainsSubset(init_not_updated, map_res)
+
         c.shutdown()
 
+        time.sleep(5)
```

Comment:
Timing can be unpredictable, we need to figure out a different way to do this. If we are waiting for  specific condition check for that condition and then continue on.